src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -191,12 +191,35 @@ private void commaSeparatedEntryTransformer(String ocrNameWithCommaSeparatedValu
         exceptionRecordToCcdFieldsMap.put(OcrFieldName.AUTHORISATION_SIGNED_BY, "authorisationSignedBy");
         exceptionRecordToCcdFieldsMap.put(OcrFieldName.AUTHORISATION_SOLICITOR_POSITION, "authorisation2b");
 
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.RESPONDENT_SOLICITOR_FIRM, "rSolicitorFirm");
+        exceptionRecordToCcdFieldsMap.put(OcrFieldName.RESPONDENT_SOLICITOR_NAME, "rSolicitorName");
+
         return exceptionRecordToCcdFieldsMap;
     }
 
-    private void applyMappingsForAddress(
-            String prefix, List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
-        applyMappings(prefix, prefix + "Address", ocrDataFields, modifiedMap);
+    private void applyMappingsForAddressExtended(
```

Comment:
Would `applyMappingsForAddress` ever be used after this is implemented ?  If not, i would think that refactoring it to account for your changes would be beneficial

Reply:
nope, it's all moved to bsp lib now.